tests/Model/Generated/Object/NotificationUrlTest.php

```diff
@@ -0,0 +1,296 @@
+            SchedulePayment::class,
+            self::GETTER_SCHEDULE_PAYMENT
+        );
+    }
+    
+    /**
+     */
+    public function testShareInviteBankResponse()
+    {
+        $this->executeNotificationUrlTest(
+            self::JSON_PATH_SHARE_INVITE_BANK_RESPONSE_MODEL,
+            ShareInviteBankResponse::class,
+            self::GETTER_SHARE_INVITE_BANK_RESPONSE
+        );
+    }
+
+    /**
+     */
+    public function testScheduledInstance()
+    {
+        $this->executeNotificationUrlTest(
+            self::JSON_PATH_SCHEDULE_INSTANCE_MODEL,
+            ScheduleInstance::class,
+            self::GETTER_SCHEDULE_INSTANCE
+        );
+    }
+
+    /**
+     */
+    public function testShareInviteBankInquiry()
+    {
+        $this->executeNotificationUrlTest(
+            self::JSON_PATH_SHARE_INVITE_BANK_INQUIRY_MODEL,
+            ShareInviteBankInquiry::class,
+            self::GETTER_SHARE_INVITE_BANK_INQUIRY
+        );
+    }
+
+    /**
+     */
+    public function testMasterCardAction()
+    {
+        $this->executeNotificationUrlTest(
+            self::JSON_PATH_MASTER_CARD_ACTION_MODEL,
+            MasterCardAction::class,
+            self::GETTER_MASTER_CARD_ACTION
+        );
+    }
+
+    /**
+     */
+    public function testChatMessageAnnouncementModel()
+    {
+        $this->executeNotificationUrlTest(
+            self::JSON_PATH_CHAT_MESSAGE_ANNOUNCEMENT_MODEL,
+            ChatMessageAnnouncement::class,
+            self::GETTER_CHAT_MESSAGE_ANNOUNCEMENT
+        );
+    }
+
+    /**
+     */
+    public function testDraftPaymentModel()
+    {
+        $this->executeNotificationUrlTest(
+            self::JSON_PATH_DRAFT_PAYMENT_MODEL,
+            DraftPayment::class,
+            self::GETTER_DRAFT_PAYMENT
+        );
+    }
+
+    /**
+     * @param string $jsonString
+     *
+     * @return NotificationUrl
+     */
+    private function getNotificationUrlFromJson(string $jsonString): NotificationUrl
+    {
+        $json = json_decode($jsonString, true);
+
+        static::assertNotNull($json, self::ASSERT_JSON_DECODE_ERROR);
+
+        $notificationObject = $this->getNotificationObjectJsonString($json);
+        $notificationUrl = NotificationUrl::createFromJsonString($notificationObject);
+        $notificationUrl = self::assertInstanceOfNotificationUrl($notificationUrl);
+
+        static::assertNotNull($notificationUrl->getObject(), self::ASSERT_OBJECT_IS_NULL_ERROR);
+
+        return $notificationUrl;
+    }
+
+    /**
+     * @param BunqModel $model
+     *
+     * @return NotificationUrl
+     * @throws BunqException
+     */
+    private function assertInstanceOfNotificationUrl(BunqModel $model): NotificationUrl
+    {
+        static::assertInstanceOf(NotificationUrl::class, $model);
+
+        if ($model instanceof NotificationUrl) {
+            return $model;
+        }
+
+        throw new BunqException(self::ASSERT_SHOULD_NOT_REACH_THIS_CODE_ERROR);
```

Comment:
The name of the error constant (just as its content) does not reflect the actual cause of the error. Please adjust it!